pkg/healthcheck/healthcheck.go

```diff
@@ -828,6 +829,20 @@ func (hc *HealthChecker) allCategories() []*Category {
 						return hc.checkPodSecurityPolicies(ctx, true)
 					},
 				},
+				{
+					description: "`clusterNetworks` config contains all node podCIDRs",
```

Comment:
Could we change the description to something like `cluster networks contain all node podCIDRs`?